src/Serilog.Sinks.Console/Serilog.Sinks.Console.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>A Serilog sink that writes log events to the console/terminal.</Description>
     <VersionPrefix>4.0.0</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>net45;netstandard1.3;netstandard2.0;netcoreapp1.1;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard1.3;netstandard2.0;netstandard2.1;netcoreapp1.1;netcoreapp2.0;netcoreapp3.0;netcoreapp3.1;</TargetFrameworks>
```

Comment:
Any reason to add the `netcoreapp3.0` and `3.1` targets since we have `netstandard2.1` covering those?

Reply:
Yes, `netstandard2.1`, `netcoreapp3.0` and `3.1`. All have the [nullable attributes](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis?view=netcore-3.0) in the API, but not the other Frameworks and old netstandard. 
I choose to add all attributes via a local file[ `src/Serilog.Sinks.Console/Support/NullableAttributes..cs`](https://github.com/serilog/serilog-sinks-console/pull/98/files#diff-6dc4d90281106151c7f8d6fce548f863) with `#if` instead of using a external depende like [Nullable](https://github.com/manuelroemer/Nullable). `netstandard2.1`, `netcoreapp3.0` and `3.1` build  should not complite this attributes, because they alredy exists.

Reply:
Now that I think about it, I think will be safe to remove `netcoreapp3.0` and `3.1` and only keep the `netstandard2.1`. I will test and remove this tomorrow.

Reply:
No, I can't remove the `netcoreapp3.0` and `3.1`, because the **PINVOKE** constant. That the `netstandard2.1` should not have but the `netcoreapp*` should.